samples/kubernetes/helm/fhir-server/templates/deployment.yaml

```diff
@@ -79,7 +80,28 @@ spec:
             - name: SqlServer__ConnectionString
               value: "Server=tcp:$(DATABASESERVERNAME),1433;Initial Catalog=$(DATABASENAME);Persist Security Info=False;User ID=$(DATABASEUSERNAME);Password=$(DATABASEPASSWORD);MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;Max Pool Size=$(MAX_POOL_SIZE)"
             {{- end }}
+            {{- if eq .Values.database.dataStore "SqlContainer" }}
+            - name: DataStore
+              value: "SqlServer"
+            - name: SqlServer__AllowDatabaseCreation
+              value: "true"
+            - name: SqlServer__Initialize
+              value: "true"
+            - name: DATABASEPASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ $dbservicename }}
+                  key: sapassword
+            - name: DATABASENAME
+              value: {{ include "fhir-server.fullname" . }}
+            - name: DATABASESERVERNAME
+              value: {{ $dbservicename }}
+            - name: SqlServer__ConnectionString
+              value: "Server=tcp:$(DATABASESERVERNAME),1433;Initial Catalog=FHIR;Persist Security Info=False;User ID=sa;Password=$(DATABASEPASSWORD);MultipleActiveResultSets=False;Connection Timeout=30"
```

Comment:
I think we should have `User Id` as a parameter. Also, there are other properties in the connection string above that we should probably include.

Reply:
Yes, the `User Id` is always `sa` for the official SQL container images, but I have made it configurable in case one might use a different image. Also allowed the database name to be changed for SQL container.

Reply:
My thinking was that you should not generally run as a user with higher privileges than you need. We will eventually be using a tool for creating the schema and can create a lower-privileged user for the FHIR server to use.